docs/content/en/docs-dev/user-guide/adding-an-application.md

```diff
@@ -7,26 +7,52 @@ description: >
 ---
 
 An application is a collect of resources and configurations that are managed together.
-It represents the service which you are going to deploy. With PipeCD, all application's manifests and its application configuration (`.pipe.yaml`) must be committed into a directory of a Git repository. That directory is called as application directory.
+It represents the service which you are going to deploy. With PipeCD, all application's manifests and its application configuration (`app.pipecd.yaml`) must be committed into a directory of a Git repository. That directory is called as application directory.
 
-Before deploying an application, the application must be registered from the web UI and a application configuration file (`.pipe.yaml`) must be committed to the application directory.
-An application must belong to exactly one environment and can be handled by one of the registered `piped`s. Currently, PipeCD is supporting the following kinds of application:
+Before deploying an application, the application must be registered from either the web UI or an application configuration file defined in your Git repository.
```

Comment:
I think both "manually form" and "git form" are registering via "web console". So maybe we don't have to explicitly distinguish them here.

```suggestion
Before deploying an application, it must be registered via the web console.
```